sormas-backend/src/main/java/de/symeda/sormas/backend/event/EventService.java

```diff
@@ -577,8 +597,53 @@ public void delete(Event event, DeletionDetails deletionDetails) {
 			actionService.deletePermanent(action);
 		}
 
-		// Mark the event as deleted
-		super.delete(event, deletionDetails);
+		event.getEventPersons().forEach(eventParticipant -> eventParticipantService.deletePermanent(eventParticipant));
+
+		sormasToSormasShareInfoService.getByAssociatedEntity(SormasToSormasShareInfo.EVENT, event.getUuid()).forEach(s -> {
+			s.setEvent(null);
+			if (sormasToSormasShareInfoFacade.hasAnyEntityReference(s)) {
+				sormasToSormasShareInfoService.ensurePersisted(s);
+			} else {
+				sormasToSormasShareInfoService.deletePermanent(s);
+			}
+		});
+
+		externalShareInfoService.getShareInfoByEvent(event.getUuid()).forEach(e -> {
+			externalShareInfoService.deletePermanent(e);
+		});
+
+		documentService.getRelatedToEntity(DocumentRelatedEntityType.EVENT, event.getUuid()).forEach(d -> documentService.markAsDeleted(d));
+
+		deleteEventInExternalSurveillanceTool(event);
+		deleteSubordinateEvents(event);
+
+		super.deletePermanent(event);
+	}
+
+	private void deleteSubordinateEvents(Event event) {
```

Comment:
```suggestion
	private void removeFromSubordinateEvents(Event event) {
```

Reply:
Reason: You're not deleting the subordinate events, you're simply removing the reference to this event